server/routes/auth.ts

```diff
@@ -237,11 +237,11 @@ passport.use(new LocalStrategy({
 			return;
 		}
 		// Send verification email (hostname validated by previous middleware)
-		let link = `http${request.secure ? "s" : ""}://${request.hostname}:${getExternalPort(request)}/auth/verify/${user.localData!.verificationCode}`;
-		let text =
-`Hi ${user.name},
+		let link = createLink(request, `/auth/verify/${user.localData!.verificationCode}`);
+		let markdown =
+`Hi {{name}},
```

Comment:
This uses the built-in email variable interpolation support which is why the `{{}}` syntax is used